docs/guides/installing-tls-certificates.md

```diff
@@ -0,0 +1,95 @@
+# Generating and installing TLS certificates
+
+- [1. Overview](#1-overview)
+- [2. Generate and install TLS certificates](#2-generate-and-install-tls-certifificates)
+   - [2.1 Generate certificates](#21-generate-certificates)
+   - [2.2 Install certificates](#22-install-certificates)
+       - [2.2.1 Install at default location](#221-install-at-default-location)
+       - [2.2.2 Install at non-default location](#222-install-at--non--default-location)
+
+## 1. Overview
+
+This document provides information about generating and installing TLS certificates
+for running infrap4d in secure mode
+
+## 2. Generate and install TLS certificates
+Note: Here target name can be `dpdk` or `es2k`
+
+### 2.1 Generate certificates
+
+- Review the files `ca.conf` and `grpc-client.conf` available under
```

Comment:
Okay, will get rid of the bullets. User can generate the certs on any system and then copy it to ACC and (server) and client can be ACC or Host.

[Response to 3] Code block was indented because it was under bullet and if you flush it to left, it doesn't look good when you view the file

[Response to 4] IP is the IP address of server

[Response to 5] Which non-example text are you referring to here?

[Response to 6] Okay, will remove the note. 

[Response to 7] copy_config_files install these generate_certs.sh under /usr/share/stratum/es2k

[Response to 8] Sabeel had developed it this, didn't want changing too many things at once.

I think we're missing some procedural context. What platform is the user running on when they invoke generate-certs.sh? The development system? The IMC? The ACC? If it's the IMC or ACC, how did they get there?
Response: Ideally they can run generate_certs.sh anywhere. Since, we provide generate_certs.sh, they can run it on Host or ACC and it will be available under /usr/share/stratum/<target_name>.

generate_cert.sh is just an example on how to generate certificates. They can use other known methods for generating TLS certificates

Reply:
> [Response to 5] Which non-example text are you referring to here?

The word "or".

Conditional variations in command-line examples are infuriatingly confusing. There are a number of these in the P4 SDE build instructions. It's easy to get confused following them. I really dislike them. Better to have a series of complete examples with differing subheadings and explanatory text.

Reply:
> [Response to 3] Code block was indented because it was under bullet and if you flush it to left, it doesn't look good when you view the file

It doesn't look good with the bullet and indented text either. The style I'm promoting is similar to the "modified block format" (for letters) that I learned in my high school typing class.